src/main/resources/application.yml

```diff
@@ -79,7 +80,7 @@ management:
         # health, info, httptrace
         # or:
         # *
-        include: ${IEXEC_CORE_MANAGEMENT_ACTUATORS:health, info}
+        include: ${IEXEC_CORE_MANAGEMENT_ACTUATORS:health, info, metrics}
```

Comment:
Do we need to expose `metrics` endpoint as a default? I'd prefer it to be exposed only if the admin wants to.

Reply:
I don't think we should make this a configurable choice. Any administrator wants to be able to monitor how the application is working.

Reply:
confusion here, this will expose `/actuator/metrics` and has nothing to do with `/metrics`

Reply:
That's it. `/actuator/metrics` will in fact expose default metrics (disk free/used/total, heap used, ...) and custom metrics ("iexec.core.workers", ...).
So, that's quite dangerous to expose it as a default. But `/metrics` is already exposed, so admin can access those values anyway :)

Reply:
do you still wish to enable `/actuator/metrics` by default ? This configuration has nothing to do with our `/metrics` endpoint which is always enabled